code/setup.dm

```diff
@@ -25,6 +25,29 @@ var/global/disable_vents     = 0
 #define CELL_VOLUME 2500	//liters in a cell
 #define MOLES_CELLSTANDARD (ONE_ATMOSPHERE*CELL_VOLUME/(T20C*R_IDEAL_GAS_EQUATION))	//moles in a 2.5 m^3 cell at 101.325 Pa and 20 degC
 
+/////GAS FLAGS & DEFINES//////
+
+#define SPECIFIC_HEAT_AIR		20
+#define SPECIFIC_HEAT_CDO		30
+#define SPECIFIC_HEAT_PLASMA	200
+#define SPECIFIC_HEAT_NIO		40
+
+#define IS_FUEL			1 //if it burns in a fire
+#define IS_OXIDISER		2 //if it acts like oxygen for a fire
+#define ALWAYS_SHOW		4 //if the scanner will record its level even if it isn't there - oxygen and nitrogen do this
+#define AUTO_FILTERED	8 //if portable scrubbers and whatnot filter it by default
+#define AUTO_LOGGING	16 //if we monitor its transfer. This is used for canisters
+
+#define OXYGEN			"oxygen"
+#define NITROGEN		"nitrogen"
+#define CARBON_DIOXIDE	"carbon_dioxide"
+#define PLASMA			"plasma"
+#define NITROUS_OXIDE	"nitrous_oxide"
+#define VOLATILE_FUEL	"volatile_fuel"

```

Comment:
these are used inappropriately in some cases but that's probably for a future PR